examples/store/src/components/product/helpers.ts

```diff
@@ -8,9 +8,13 @@ export type MedusaProduct = Awaited<
     ReturnType<Medusa["products"]["list"]>
 >["products"][number];
 
+// eslint-disable-next-line
 export function getProductVariant(product: any, opts: SelectedOptions) {
+    // eslint-disable-next-line
     const variant = product?.variants.find((variant: any) => {
+        // eslint-disable-next-line
         return Object.entries(opts).every(([key, value]) =>
+            // eslint-disable-next-line
             variant.options.find((option: any) => {
                 return (
```

Comment:
```suggestion
export function getProductVariant(product: Product, opts: SelectedOptions) {
    const variant = product?.variants.find((variant) => {
        return Object.entries(opts).every(([_key, value]) =>
            variant.options.find((option) => {
                return (
```

Sorry, that's a mistype on our end, we could have typed the `product` parameter as `Product` from `medusa`, this will solve the `any` types.